includes/helper/class-wp-job-manager-helper-api.php

```diff
@@ -168,6 +168,12 @@ protected function request( $args, $return_error = false ) {
 			'email'          => '',
 		];
 
+		// These legacy endpoints are temporary. For now, translate `license_key` => `licence_key` at this point.
+		if ( ! empty( $args['license_key'] ) ) {
+			$args['licence_key'] = $args['license_key'];
+			unset( $args['license_key'] );
+		}
```

Comment:
Very minor thing, but I think it makes more sense to handle this "translation" first thing in the method.

Currently `$defaults` are defined above and they reference the old name (`licence_key`).

Reply:
Moved in 72413d56682e60d9ce555e4cb1c15e97ad1fe879